src/test/ui/consts/const_let_assign2.rs

```diff
@@ -1,6 +1,5 @@
 // compile-pass
 
-#![feature(const_let)]
 #![feature(const_fn)]
```

Comment:
Not immediately seeing what requires `const_fn` here... are we not stabilizing assignment to projections perhaps?